File: Problem 1828.c
#include<stdio.h>
#include<string.h>
int main()
{
    int i,j,k,l,m,n;
    char ch[10],ch1[10];
    scanf("%d",&n);
    for(i = 1;i <= n;i++){
        scanf("%s %s",&ch, &ch1);
        if(!strcmp(ch, ch1))printf("Caso #%d: De novo!\n",i);
        else if(!strcmp(ch, "lagarto")){
            if(!strcmp(ch1, "Spock") || !strcmp(ch1, "papel"))printf("Caso #%d: Bazinga!\n",i);
            else if(!strcmp(ch1, "tesoura") || !strcmp(ch1, "pedra"))printf("Caso #%d: Raj trapaceou!\n",i);
        }
        else if(!strcmp(ch, "tesoura")){
            if(!strcmp(ch1, "lagarto") || !strcmp(ch1, "papel"))printf("Caso #%d: Bazinga!\n",i);
            else if(!strcmp(ch1, "Spock") || !strcmp(ch1, "pedra"))printf("Caso #%d: Raj trapaceou!\n",i);
        }
        else if(!strcmp(ch, "papel")){
            if(!strcmp(ch1, "Spock") || !strcmp(ch1, "pedra"))printf("Caso #%d: Bazinga!\n",i);
            else if(!strcmp(ch1, "lagarto") || !strcmp(ch1, "tesoura"))printf("Caso #%d: Raj trapaceou!\n",i);
        }
        else if(!strcmp(ch, "Spock")){
            if(!strcmp(ch1, "tesoura") || !strcmp(ch1, "pedra"))printf("Caso #%d: Bazinga!\n",i);
            else if(!strcmp(ch1, "lagarto") || !strcmp(ch1, "papel"))printf("Caso #%d: Raj trapaceou!\n",i);
        }
        else if(!strcmp(ch, "pedra")){
            if(!strcmp(ch1, "tesoura") || !strcmp(ch1, "lagarto"))printf("Caso #%d: Bazinga!\n",i);
            else if(!strcmp(ch1, "Spock") || !strcmp(ch1, "papel"))printf("Caso #%d: Raj trapaceou!\n",i);
        }
    }
    return 0;
}

File: Problem 1828.py
for i in range(int(input(""))):
    j1, j2 = input("").split()

    if j1 == j2:
        win = "De novo!"
    elif j1 == "pedra":
        if j2 == "tesoura" or j2 == "lagarto":
            win = j1
        else:
            win = j2
    elif j1 == "papel":
        if j2 == "pedra" or j2 == "Spock":
            win = j1
        else:
            win = j2
    elif j1 == "tesoura":
        if j2 == "lagarto" or j2 == "papel":
            win = j1
        else:
            win = j2
    elif j1 == "lagarto":
        if j2 == "Spock" or j2 == "papel":
            win = j1
        else:
            win = j2
    elif j1 == "Spock":
        if j2 == "tesoura" or j2 == "pedra":
            win = j1
        else:
            win = j2

    if win == j1:
        win = "Bazinga!"
    elif win == j2:
        win = "Raj trapaceou!"

    print("Caso #%i:" %(i + 1), win)